pkgs/applications/editors/vscode/update-vscode.sh

```diff
@@ -16,8 +16,7 @@ fi
 
 # VSCode
 
-VSCODE_VER=$(curl -s -L "https://code.visualstudio.com/Download" | grep "is now available" | awk -F'</span>' '{print $1}' | awk -F'>' '{print $NF}')
-VSCODE_VER=$(curl -s -L "https://code.visualstudio.com/updates/v${VSCODE_VER/./_}" | grep "Downloads:" | awk -F'code.visualstudio.com/' '{print $2}' | awk -F'/' '{print $1}')
+VSCODE_VER=$(curl --silent https://api.github.com/repos/Microsoft/vscode/releases/latest | jq --raw-output .tag_name)
```

Comment:
Would be nice to add `curl --fail ...` too.

Reply:
Oh, good point @bjornfor ! Fixed in edfc4d1.